docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -169,23 +174,23 @@ To disable TC completely, use this setting in your project file:
 
 You can improve the startup time of your .NET Core application by compiling your application assemblies as ReadyToRun (R2R) format. R2R is a form of ahead-of-time (AOT) compilation.
 
-R2R binaries improve startup performance by reducing the amount of work the just-in-time (JIT) compiler needs to do as your application loads. The binaries contain similar native code compared to what the JIT would produce.
+R2R binaries improve startup performance by reducing the amount of work the just-in-time (JIT) compiler needs to do as your application loads. The binaries contain similar native code compared to what the JIT would produce. However, R2R binaries are larger because they contain both intermediate language (IL) code, which is still needed for some scenarios, and the native version of the same code. R2R is only available when you publish a self-contained app that targets a specific runtime environments (RID) such as Linux x64 or Windows x64.
 
-R2R binaries are larger because they contain both intermediate language (IL) code, which is still needed for some scenarios, and the native version of the same code. R2R is only available when you publish a self-contained app that targets a specific runtime environments (RID) such as Linux x64 or Windows x64.
+To compile your project as ReadyToRun, do the following:
 
-To compile your app as R2R, add the `<PublishReadyToRun>` setting:
+01. Add the `<PublishReadyToRun>` setting to your project
 
-```xml
-<PropertyGroup>
-  <PublishReadyToRun>true</PublishReadyToRun>
-</PropertyGroup>
-```
+    ```xml
+    <PropertyGroup>
+      <PublishReadyToRun>true</PublishReadyToRun>
+    </PropertyGroup>
+    ```
 
-Publish a self-contained app. For example, this command creates a self-contained app for the 64-bit version of Windows:
+01. Publish a self-contained app. For example, this command creates a self-contained app for the 64-bit version of Windows:
```

Comment:
@mungojam Can you review this section to make sure that it's clearer? I changed the format to a numbered list, showing that you do these two things to enable R2R.